TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -281,6 +281,28 @@ private DatetimeConstant CreateDatetimeConstant(CodeElementsParser.Datetime_cons
                 return new DatetimeConstant(literal, DatetimeConstantKind.Timestamp, (Token)context.timestamp().KeywordTIMESTAMP);
             }
 
+            return null;
+        }
+        private HostVariable CreateSqlHostVariable(CodeElementsParser.HostVariableContext context)
+        {
+            if (context.mainVariable != null)
+            {
+                Token mainVariableToken = (Token) context.mainVariable;
+                SymbolReference mainSymbolReference = new SymbolReference(
+                    new AlphanumericValue(mainVariableToken), SymbolType.SqlVariable);
+                if (context.indicatorVariable != null)
+                {
+                    Token indicatorVariableToken = (Token)context.indicatorVariable;
+                    SymbolReference indicatorVariableSymbolReference = new SymbolReference(
+                        new AlphanumericValue(mainVariableToken), SymbolType.SqlVariable);
```

Comment:
No. You are using `mainVariableToken` instead of `indicatorVariableToken`, be careful when copy/pasting !!